gremlin/templates/daemonset.yaml

```diff
@@ -83,6 +83,7 @@ spec:
         resources: {{ toYaml .Values.gremlin.resources | nindent 10 }}
         {{- end }}
         securityContext:
+          allowPrivilegeEscalation: {{ .Values.gremlin.podSecurity.allowPrivilegeEscalation }}
```

Comment:
already defined and set to `false` in `values.yaml`. This is just plumbing that value through to the daemonset.